lib/beaker/host/unix.rb

```diff
@@ -138,6 +138,23 @@ def external_copy_base
       @external_copy_base
     end
 
+    # Tells you whether a host platform supports beaker's
+    #   {Beaker::HostPrebuiltSteps#set_env} method
+    #
+    # @return [String,nil] Reason message if set_env is not supported,
+    #   nil if it is supported
+    def supports_set_env
```

Comment:
Walls of text are my preferred form of communication   :)


(srsly, you can give _what_ you think concisely, but I find it much harder to concisely explain _why_ you think that way, and understanding the why is the interesting part of any discussion)